src/panels/config/devices/device-detail/integration-elements/zwave_js/ha-device-info-zwave_js.ts

```diff
@@ -110,18 +139,22 @@ export class HaDeviceInfoZWaveJS extends SubscribeMixin(LitElement) {
             )}:
             ${this._node.node_id}
           </div>
+          <div>
+            ${this.hass.localize(
+              "ui.panel.config.zwave_js.device_info.node_status"
+            )}:
+            ${this.hass.localize(
+              this._node.is_controller_node
+                ? `ui.panel.config.zwave_js.controller_status.${
+                    controllerStatus[this._controller!.status]
```

Comment:
Use the enum for this

Reply:
won't that return the enum value not the name?